File: src/main/scala/io/github/h2sxxa/scala3std/fabric/Scala3Std.scala
package io.github.h2sxxa.scala3std.fabric

import cross.net.fabricmc.api.ModInitializer
import io.github.h2sxxa.scala3std.common.LibDetector

object Scala3Std extends ModInitializer {
  override def onInitialize(): Unit = LibDetector.scan()
}

File: src/main/java/io/github/h2sxxa/scala3std/fabric/Scala3Std.java
package io.github.h2sxxa.scala3std.fabric;

import cross.net.fabricmc.api.ModInitializer;
import io.github.h2sxxa.scala3std.common.LibDetector;
import scala.collection.immutable.List;

import java.util.Properties;

@SuppressWarnings("unused")
public class Scala3Std implements ModInitializer {
    public final static List<Properties> properties = LibDetector.scan();

    @Override
    public void onInitialize() {
    }
}
